addons/dragon/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\ace\addons\dragon
```

Comment:
EOF

Reply:
Not in pboprefix. Some packers take full file, which would cause errors.
None of ACE prefixes has empty line.